packages/services/api/src/modules/organization/providers/organization-manager.ts

```diff
@@ -239,15 +239,19 @@ export class OrganizationManager {
 
   @cache((selector: OrganizationSelector) => selector.organizationId)
   async getInvitations(selector: OrganizationSelector) {
-    await this.session.assertPerformAction({
+    const canAccessInvitations = await this.session.canPerformAction({
       action: 'member:modify',
       organizationId: selector.organizationId,
       params: {
         organizationId: selector.organizationId,
       },
     });
 
-    return this.storage.getOrganizationInvitations(selector);
+    if (!canAccessInvitations) {
+      return null;
+    }
+
+    return await this.storage.getOrganizationInvitations(selector);
```

Comment:
This is a pragmatic solution to fix this problem. 
Alternatively, we could ignore the error on the frontend, but that would require more logic there for error handling.

Thinking of the upcoming public API, we might even want to have a "throw" field for the public API, but a "just return null" field for our dashboard API...

Reply:
I added a reference for this to https://www.notion.so/theguildoss/Hive-GraphQL-API-Design-Patterns-18ab6b71848a80678939c669699fe8e6?pvs=4#1a7b6b71848a807aa645eee88f98c93a